src/source/release-notes.rst

```diff
@@ -0,0 +1,194 @@
+.. index:: Release Notes
+.. _release_notes:
+
+Release Notes
+=============
+
+This section attempts to give a little more insight on what our various releases contain, as compared to previous versions.
+
+Feb 2023: 'Campaigns & Micros'
+------------------------------
+
+Synopsis
+""""""""
+
+* For bid, budgets & spent values we now report in micros: 1 cent == 10000 micros. These fields will now also be named accordingly, mentioning whether they represent a value in cents or micros. This is also represented in the categories data, where the interval of allowed values is defined. Mind, this does not include values like 'AUTOMATIC' or 'UNLIMITED'.
```

Comment:
> Mind, this does not include values like 'AUTOMATIC' or 'UNLIMITED'.
Can you expand on this? I got confused for a second on what _this_ refers to

Reply:
hm, we have fields called 'bidMicros', so in a way, yes it will be introduced in this version. I can make an explicit piece about that re-naming.
I'll also rephrase this section to make it clearer.